vtkext/private/module/vtkF3DEXRReader.h

```diff
@@ -44,6 +56,52 @@ class vtkF3DEXRReader : public vtkImageReader
 private:
   vtkF3DEXRReader(const vtkF3DEXRReader&) = delete;
   void operator=(const vtkF3DEXRReader&) = delete;
+
+  /**
+   * Class to treat file contents in memory like it were still in a file.
+   */
+  class MemStream : public Imf::IStream
```

Comment:
Let's move this class in the cxx file to avoid any need for openexr include in the header.